.vscode/settings.json

```diff
@@ -32,7 +32,7 @@
 	},
 	"[typescriptreact]": {
 		"editor.rulers": [80],
-		"editor.defaultFormatter": "esbenp.prettier-vscode",
+		"editor.defaultFormatter": "vscode.typescript-language-features",
```

Comment:
did someone tell you to change this

Reply:
no, It was just changed after I ran the prettier thing I think?